python/torch_mlir_e2e_test/test_suite/elementwise_comparison.py

```diff
@@ -578,6 +578,48 @@ def ElementwiseNeIntScalarModule_basic(module, tu: TestUtils):
 
 # ==============================================================================
 
+class ElementwiseNeFloatTensorModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([2, 3], torch.float32, True),
```

Comment:
Can you also add a test case with dynamic shape?

Reply:
Added, but because it will be failed with dynamic shape I will not put it in the PASS_SET right now